crates/sui-verifier/src/id_leak_verifier.rs

```diff
@@ -147,17 +124,15 @@ impl<'a> IDLeakAnalysis<'a> {
 
 impl<'a> TransferFunctions for IDLeakAnalysis<'a> {
     type State = AbstractState;
-    type AnalysisError = ExecutionError;
 
     fn execute(
         &mut self,
         state: &mut Self::State,
         bytecode: &Bytecode,
         index: CodeOffset,
         _: CodeOffset,
-    ) -> Result<(), Self::AnalysisError> {
-        execute_inner(self, state, bytecode, index)?;
-        Ok(())
+    ) -> PartialVMResult<()> {
```

Comment:
@sblackshear all of these had to become `PartialVMResults` because of the simplifications to the abstract interpreter last weekend. Not a big deal, since all of these errors get collapsed into one bucket anyway, but worth pointing out, we might want to bring the associated type back :)

Reply:
Sorry for that accidental breakage... Yeah, the associated type does seem useful for bytecode verifier extensions. Didn't think about that.